src/components/QuestCard/index.tsx

```diff
@@ -44,7 +44,8 @@ export function QuestCard({
   classNames,
   selected,
   ...rest
-}: QuestCardProps) {
+}: QuestCardProps &
+  Omit<React.ComponentPropsWithoutRef<'div'>, keyof CardGenericProps>) {
```

Comment:
we should update the `QuestCardProps` type to include this

Reply:
nice job getting the div props + the mantine type. I missed that on some components

Reply:
Updated, also FYI in case you would like to check, I've removed `className` as seems we are not using it at all, while we've got `className.root` which does the same functionality 

Effected Components:

`AchievementsCard`
`QuestCard`
`GenericCard`